presto-hive/src/main/java/io/prestosql/plugin/hive/s3/PrestoS3FileSystem.java

```diff
@@ -238,6 +242,7 @@ public void initialize(URI uri, Configuration conf)
         String userAgentPrefix = conf.get(S3_USER_AGENT_PREFIX, defaults.getS3UserAgentPrefix());
         this.skipGlacierObjects = conf.getBoolean(S3_SKIP_GLACIER_OBJECTS, defaults.isSkipGlacierObjects());
         this.requesterPaysEnabled = conf.getBoolean(S3_REQUESTER_PAYS_ENABLED, defaults.isRequesterPaysEnabled());
+        this.s3StorageClass = PrestoS3StorageClass.valueOf(conf.get(S3_STORAGE_CLASS, defaults.getS3StorageClass().name()));
```

Comment:
Can we use `conf.getEnum` to directly fetch the enum ?

Reply:
- This is consistent with `PrestoS3AclType.valueOf(...` above.
- The additional logic applied in `org.apache.hadoop.conf.Configuration#getEnum` (ie trimming) is not useful here, since the value comes from our code, and not from configuration.

Let's leave as is. (Or change both in follow up)